satpy/etc/readers/aws1_mwr_l1b_nc.yaml

```diff
@@ -0,0 +1,535 @@
+reader:
+  name: aws1_mwr_l1b_nc
```

Comment:
What is the `1` in `aws1`? I thought there'd ever be only one AWSat, and the next ones would be named "Sterna".

Reply:
Yes, I am not really very happy about this. But those in charge claied you can't have a satellite with only three letters, thus AWS1, I have communicated this with EUMETSAT and ESA who has the responsibility towards WMO (Oscar). 
So, satellite name = "AWS1"
Instrument name = "MWR" as well as "EPS-Sterna Radiometer".
The latter came up while the radiomete name was still undecided at ESA/EUMETSAT side, and thus started being used. Think it is already in use in some BUFR tables, so we are stuck with two names for the radiometer! :-(

Reply:
:facepalm: